### DISCUSSION TITLE
레벨1 회고 | 2022.04.08

### DISCUSSION THREAD

**Discussion by wmakerjun** [woowacourse/retrospective#12] [General]
# 레벨1 회고
### 1. 아래 시트를 복사해서레벨1 보이는 라디오 팀원들과 함께 회고를 진행합니다.
https://docs.google.com/spreadsheets/d/10RSrjq3bRLxILugll8lxyuS-V1yJJ02WC41iNdqvbsg/edit?usp=sharing

### 2. 회고를 마무리한 후 디스커션 코멘트에 팀별로 '회고에 대한 회고' 내용과 '스프레드 시트 링크'를 남깁니다. 
- view 권한 체크 후 남겨주세요!

**Comment by her0807** [comment]
## 에티켓의 민족 회고

``` java
// 백엔드 회고 내용
회고가 좋다.. 
void levelOne() {
    if (perfect) {
        return;
    }
    levelOne();
}


// 프론트 회고 내용
const 좋았다 = true;


```

- [회고 내용](https://docs.google.com/spreadsheets/d/1sEhi2FeSXoTmnTNNy8kZ2kWBR6ANXuw7HoOpllPDxRA/edit#gid=0)

**Reply by Ohzzi** [reply]
선생님 perfect가 변경되지 않습니다

**Reply by jayjaehunchoi** [reply]
시적허용이라 생각해주세요

**Reply by yeon-06** [reply]
완벽할때까지 레벨1을 한다 와..

**Reply by Ohzzi** [reply]
Value 'perfect' is always 'false'

**Reply by jayjaehunchoi** [reply]
하남자특 ) 이악물고 달려듬

**Comment by awesomeo184** [comment]
[회고링크](https://docs.google.com/spreadsheets/d/1PdCTzcJLlQ1r0uE88ltplUj3-Cl2QEE5AbYkP9-xal0/edit#gid=0)


* 어썸오 - 레벨 1 나름대로 만족하면서 마치네요. 보이는 라디오 조 오랜만에 만나서 너무 반가웠어요~

* 주디 -  leve1이 벌써 끝나는게 너무 아쉽네요! 아직 많이 부족하지만 열심히 했으니 이만하면 된 것 같기도..? 오늘 오랜만에 보라팀과 만나서 너무 좋았어요 :)

* 칙촉 - 내일은 푹자야지~ 우벤져스로 만난지 벌써 2개월이 지났다니. 시간 참 빠르네요. 오랜만에 만나서 반가웠어요!! 모두 방학 잘 보내세요!!!!

* 승팡 - 벌써 레벨1이 끝난다니.... 레벨2에는 더 발전된 모습!

* 차리 - 오늘 벤치 1rm 잰다( 운동안한지 2주됨 )

* 나인 - 레벨1이 벌써 끝났다니 시간이 정말 빠르네요. 레벨1의 좋은 기억들을 가지고 레벨2도 뿌셔봅시다~!

* 민초 - 레벨 1의 시작과 끝을 보라조와 함께 할 수 있어서 좋았습니다!! 레벨 1 기간 동안 고생하셨고, 남은 레벨도 다들 화이팅하시길 바랄게요!!

* 헌치 - 성장과 건강의 극단적 교환이 있었네요. 저도 이제부터 운동하겠습니다^^ 우벤저스 포에버~

**Comment by sure-why-not** [comment]
## 가짜우테코 회고
### “야호~ 신난다, 방학이다~”
![image](https://user-images.githubusercontent.com/94965944/162353169-ede36f0e-9987-4250-83a0-21a78eccd171.png)

- [회고](https://docs.google.com/spreadsheets/d/16gY0OSCJBAsaREytZSHjvaMVZ69_n0hSPS-sBDg4fgM/edit#gid=0)

**Reply by moonheekim0118** [reply]
무야호~~~~~

**Reply by prefer2** [reply]
와~ 방학이다~~

**Reply by gudonghee2000** [reply]
이야~~~

**Reply by jaejae-yoo** [reply]
야호 ~~~ 방학이다 ~~

**Reply by Ohzzi** [reply]
신난다~~ 무야호~~

**Comment by ulimy** [comment]
[우토피아 회고 시트](https://docs.google.com/spreadsheets/d/1I4jgohcUW-ePeuyk23ST1Jvx9AdGPiCHULGOsP_IhJY/edit?usp=sharing)

아리 - 마지막에 우토피아 팀원들을 다시 만날 수 있어서 너무너무 좋았다!!! 앞으로도 종종 연락하며 지내고싶다 :)

디우 - 우테코 시작을 함께한 보이는 라디오 조 크루들을 만나 회고를 하니 너무 반갑고, 자주 이렇게 만나도 좋을 것 같다!

베루스 - 마지막에 보이는 라디오 조와 회고를 할 수  있어서 좋았다. 자주 볼 수 있으면 좋을 것 같다!

조조그린 - 다음엔 더 좋은 컨디션에서 회고를 해야겠다.

록바 - 오랜만에 우토피아조를 만나서 좋다. 회고를 하니, 눈물이 난다.

빅터 - 오랜만에 만나서 너무나 좋았다!

**Reply by lokba** [reply]
우토피아 짱

**Reply by ulimy** [reply]
동물 개발자님들 다시 만나서 반가웠습니다^~^

**Reply by jojogreen91** [reply]
우토피아 모두 수고하셨습니다😀

**Reply by tco0427** [reply]
레벨1 다들 고생하셨어요~_~

**Comment by meatsby** [comment]
## 방학 계획

### ㅋㅋ 이것만 보고 공부해야징
![47688989_2234879966795488_3277828519576469504_n](https://user-images.githubusercontent.com/83967672/162354337-1b8bb19f-0dfd-400e-a574-f35c5c565154.jpeg)

### 아 몰랑 내일 할래
![47683127_2234880006795484_870984982120628224_n](https://user-images.githubusercontent.com/83967672/162354437-aa90adf4-286d-4368-b30c-37d6b2dfe3d6.jpeg)

[즐거운 코딩 회고 ^오^](https://docs.google.com/spreadsheets/d/10I5wwHorMo6LgLW_vLzfV15q_H1ofmBwFknLoP_18ug/edit?usp=sharing)

**Reply by pocojang** [reply]
권한 막혀있네요 👀

**Comment by YJGwon** [comment]
### 18조 아이스에이지 회고

[스프레드 시트](https://docs.google.com/spreadsheets/d/1f5os2mAQ-aZmDWtx1TTUfb3y7VIa1-QgF14A_3KGPos/edit?usp=sharing)

![결](https://ifh.cc/g/9pytbt.jpg)
사라진 조원을 찾습니다. 회식 재밌으셨나요..?

**Reply by ronci** [reply]
저희 조원 공개수배합니다.

**Reply by YJGwon** [reply]
잠실에서 1심. 선릉에서 2심. 재판 딱대.

**Reply by yhh1056** [reply]
<img width="644" alt="스크린샷 2022-04-08 오후 12 05 42" src="https://user-images.githubusercontent.com/58363663/162354995-cb9cf93a-e07c-47da-8498-25969f98cf07.png">

**Reply by ronci** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by Hongdonggeon** [reply]
![결](https://user-images.githubusercontent.com/67885363/162355826-9b961847-8fa6-46ca-b6ba-1f0b214004ea.jpg)

**Reply by YJGwon** [reply]
결.....저희는 쿤 말렸어요 ㅎ

**Reply by Hongdonggeon** [reply]
????!!!!?!!?!??!!?

**Reply by ronci** [reply]
결 저희는 지키려고 노력했습니다... 모든건 쿤의 독단적인 행동으로..

**Reply by YJGwon** [reply]
아이스에이지 공식입장은 아닙니다. 쿤 개인의 의견으로...

**Comment by kbsat** [comment]
[회고](https://docs.google.com/spreadsheets/d/1WO7igEz521Pnoyr7aGomHL24BfedGNKXdUu_SnrVD6M/edit#gid=0)

### 열음이었다..

<img width="2129" alt="스크린샷 2022-04-08 12 05 10" src="https://user-images.githubusercontent.com/52696169/162354935-28c07aa9-c750-4d61-b59a-e5f4417d0ecc.png">

**Reply by bugoverdose** [reply]
찢었다

**Comment by soyi47** [comment]
### [순수한 반란군들 Level 1 회고](https://docs.google.com/spreadsheets/d/12f82lTgg6NrDr5793TFZStCh7GNali95EzqI3BqAeUQ/edit#gid=0)

#### 회고에 대한 회고

아스피 - 보이는 라디오 역시 재밌네요 ㅎㅎ

소피아 - 이 사람들 수다 참 잘 떤다... 그냥 수다 타임 ㅎㅅㅎ

배카라 - 보이는 라디오 너무 재밌어요

아놀드 - 보이는 라디오 조와 만나서 행복해요~

조시 - 다음주 회식이 너무 기대되네요!

기론 - 다들 술을 잘 마셔서 무섭네용

#### 조원들에게 마지막 한 마디

아스피 - 빨리 13일에 보고싶다....

소피아 - 곧 봅시다~~

배카라 - 킹갓 아놀드

아놀드 - 배카라 나 가지고 논거야..?

조시 - 자주봐요!!!

기론 - 자주 만나요~.~

**Reply by summerlunaa** [reply]
아스피 또 취하시려고?....

**Comment by kimchan123** [comment]
[**개복치조 회고**](https://docs.google.com/spreadsheets/d/11aQtoLF0YV8fMBBAF31F4ZzBcwJUz8n_uSZ8xxn6--Q/edit?usp=sharing)

엘리 : "역시 회고는 하기전에는 뭘 말해야할지.. 걱정되지만 
막상 하고나면 마음이 정리되면서 도움이 많이 된다❤️ 
근데 왜 항상 시작하기 전에는 하기 싫은걸까..?😅"

찬 : "오랜만에 프론트엔드 분들이랑 얘기할 수 있어서 너무 반가웠다. 
레벨2가 기대된다. 방학이다~~~~~"

짱구 : "보이는 라디오 크루들과 이렇게 다시 만나게 되어 너무 반갑고 행복하다!
다들 그때보다 훨씬 우테코에 적응한 것 같아서 안심이다.
레벨 2에도 지치지 않고 모두 즐거운 개발 인생 이어나가면 좋겠다."

애쉬 : "보이는 라디오 조와 다시 만나게 되어 반가웠다.
프론트 분들의 생각도 들을 수 있어서 좋았고 프론트엔드의 게더타운은 신기했다.
다시한번 레벨2의 각오를 다져봐서 좋았다. 방학동안 레벨1을 잘 정리해서 레벨2로 가보자"

콤피 : "백엔드분들의 근황과 생활을 들을 수 있어서 신기했다.
백엔드는 ""야생""인것 같다..ㅋㅋㅋㅋㅋㅋㅋ"

유세지 : "백엔드 분들의 근황도 들을 수 있어서 알찼던 회고 시간이었다.
프론트도 힘들다고 생각했는데 백도 장난 아닌 것 같다... 🔥
조금 더 성장해서 백엔드 분들과 프로젝트도 해보고 싶다!"

**Reply by usageness** [reply]
두근두근

**Comment by EastHShin** [comment]
[회고 링크](https://docs.google.com/spreadsheets/d/1eE4T9ByO3OR_GkXXh2AzzWTH1Qdh-ElnEzqXv881WJU/edit#gid=0)

페퍼 - 보라 조 오랜만에 보는데 근황 토크한거 같아 좋았습니다! 다들 열심히 살고 계셔서 자극받고 갑니다. 레벨투때는 더 화이팅! 오프라인에서 만나요 여러분~~

이스트 - 레벨1에서 느낀 아쉬움을 레벨2에서 고쳐나갔으면 좋겠다! 오랜만에 보라조 만나서 즐거웠다!!

꼬재 - 다른 크루들도 아쉬움이 많이 남은 level1이였던 것 같다..! level2에서는 level1에 아쉬웠던 점을 잘 보완해서 달려보장!!

바니 - 시간 관리가 잘 되지  않았던  레벨 1이다. 레벨  2는 정신차리자 제발~~

앤지 - 너무 아쉽다 ㅠ^ㅠ 레벨 2때 정신 차릴 사람 추가요~! 보라 조 용사분들과 오랜만에 이야기를 나눠서 행복했다 :) 

우디 - 우디 언제와~~

**Reply by kkojae91** [reply]
배고프다!!!!!!

**Comment by leo0842** [comment]
[회고 링크](https://docs.google.com/spreadsheets/d/1V68_Wgn49d5hC7LelFAdO1M2N_XklH4RN1NOd_NimrE/edit#gid=0)

에덴 - 다들 오랜만에 만나서 좋았다. 오프라인하면 더 자주 보게 될테야 배가 살짝 아픔

토르 - 우테코 와서 처음 친해진 사람들이라 근본이 있네요. 오프 때 뵙죠

자스민 - 사람좀 더 많이 보고싶다! 방학때 어떤걸할지 고민해야겠다..

후이 - 처음 볼때는 서로 서먹했었는데, 적응하고 만나니까 훨씬 재밌었다

제로 - 배려있는 페어들을 많이 만날 수 있었고 많이 배울 수 있어서 좋았다. 혼자 살 때 몸이 아프면 서럽다는 것도 직접 느꼈다. 방학때 많이 쉬고 레벨2때 다시 화이팅하자.

써머 - 오랜만에 보라조 봐서 좋았다 대면이 기대됨  감점사유 : 여전히피곤함

![image](https://user-images.githubusercontent.com/42317507/162355897-728dc244-b693-4f24-9925-9fdb79b0e09f.png)

**Comment by DWL21** [comment]
[13조 회고](https://docs.google.com/spreadsheets/d/1Kj1GyL0HGLfdj_4h2xpzeW_syds9M9RehnfwvXYuokc/edit?usp=sharing)

레오: 과정이 끝날 때 회고를 하니 레벨 하나가 끝났다는게 실감이 난다.
필즈: 확실히 무엇을 할 지에 대해 정리를 해보는 것은 좋은 것 같다. 메타인지가 되었다.
판다: 시작할때는 레벨1이 길어보였는데 이게 벌써 끝나네. 레벨 2 시간을 낭비하지 않으려면 좀 더 열심히 살아야 할듯
루나: 오랜만에 본 라디오팀이랑 코드도 같이 보고 수다 떨었더니 재밌게 시간 보낸 것 같다. 우리 오프라인때도 같이 힘내봐요!
비녀: 오랜만에 백엔드 얼굴 보니까 좋네요. 같이 lv1 마지막 과제 뜯어보니까 더 새로운느낌
온스타: 오랜만에 반가운 사람들과 이야기를 나눌 수 있어서 좋았다. 무엇을 해야할지 막막했고 걱정이 많았던 시기에 만났던 사람들이었는데, 이제는 서로 무엇을 해야할지 알고, 그에 대한 솔루션에 대해서도 이야기하니 기분이 좀 멜랑꼴리하다.

**Comment by wonsss** [comment]
### [12조 회고](https://docs.google.com/spreadsheets/d/1-mapbbhw11KMGJAcAeKLWMp3znCny7sdlxmdl37Bjfg/edit?usp=sharing)
- 동키콩: "보라조 내마음 속에 저장~~ 마르코, 이프, 스컬, 렉스, 슬로 사랑합니다!!"
- 이프 : "지난날을 함께했던 쿠키런 크루들과 다시 소통할 수 있게 되어 행복하다. 레벨1에 대한 회고를 공유하니, 서로 간의 관계가 더욱 친밀해졌다. 오프라인에서의 보이는라디오 회식이 벌써부터 기대된다. 와 방학이다~"
- 스컬 : 몸은 여전히 피곤하지만 그래도 여기가 천국이야😍
- 렉스 : "오랜만에 보이는 라디오 조 크루들과 이야기를 하니 반가웠습니다. 종종 만나면 좋겠습니다"
- 마르코 : "레벨1이 끝나는 시점에서, 레벨1 시작을 함께했던 보라조를 다시 만나게 되니, 처음에 가졌던 생각이 다시 떠올랐다. 그 설레던 마음 그대로 열심히 해야겠다. 편안한 분위기에 함께 편안히 이야기해서 좋았어요."
- 슬로 : "- 오랜만에 보이는 라디오 크루분들을 볼 수 있어서 좋았다.레벨1 기간동안 부족했던 부분, 방학동안 보완할 것을 정리할 수 있어서 좋았다."

**Comment by devHudi** [comment]
## [9조 회고](https://docs.google.com/spreadsheets/d/1kpAVFb4cYaiTeMRudR2NBRgNV6cHv5FU1OVcCZOWC2A/edit#gid=0)

- `토닉` : 오랜만에 보이는라디오 조원들과 얘기해서 좋았어용~ 레벨1을 회고하면서 마음이 정리되어 좋았습니다!
- `후디` : 보라조하고 이렇게 다시 만나게 될 줄 몰랐는데, 너무 반갑고 다들 각자의 위치에서 재밌게 열심히 생활하시는 것 같아서 좋은 자극 많이 받아갑니다 ㅎㅎㅎㅎ
- `돌범` : 다시금 인텔리제이에서 벗어나 이 사람들과 얘기할 시간을 만들어준 것에 감사드립니다.
- `준찌` : 오랜만에 친목도 다지고 서로 좋은말도 주고받다보니 굉장히 힐링되네요 !!
- `루키` : 오랫만에 보라조랑 얘기하니깐 좋네요!! 다들 열심히 레벨 2 해나갔으면 좋겠습니다!!
- `블링` : 수미상관으로 처음과 끝을 같이하는 보라조 너무 좋습니다! 이렇게 백엔드 크루들과 소통하는 기회가 더더 많아졌으면 좋겠네용

**Comment by yaho99** [comment]
[6조 회고 링크](https://docs.google.com/spreadsheets/d/17azGgpyeu4y5eQvip6dv1wGXAeFxXkhhna3b-ckUJ60/edit#gid=0)

티키 : 오랜만이라 반갑네요 회고를 이제 정말 방학이라는게 너무 좋네요

야호 : 오랜만에 보이는 라디오 멤버들을 만나서 너무 좋네요~!~!특히 프론트 분들을 오랜만에 봐서 좋았습니다 😎
 
하리 : 회고 하다보니까 기분이 좋아졌다~ 보라 조원들 최고 👊 오랜만에 만나서 너~무 좋았다!

무비 : 후,, 오랜만에 만난 보라멤버들,, 아련하게 반갑다. 다들 잘 지내셨죠? 🥂 저는 대부분 못지냈습니다. 
호호 저와 하리는 자판기를 수리하고 왔구요,, 자판기공이 되어버렸습니다. 아무튼 역시 회고를 하니,, 마음이 따땃하네요.. 회고는..국밥이다 ☕️

알파 : 사실 회고를 한다고 크게 달라지는 건 없지 않나라고 생각했던 과거였으나 무엇이 부족한지를 깨닫는 시간이라서 좋았다. 차후의 데이터로서 이용할 수 있는 기회라고 생각된다. 앞으로의 회고도 열심히 해야지

알린 : 아침부터 웃으니 참 행복하다 지난 두달이 생각나네요 좋아요 😀

**Reply by OzRagwort** [reply]
좋았습니다

**Reply by yh20studio** [reply]
잠실에서 만날지도?

**Reply by byhhh2** [reply]
우리 의무적인..보라조..☕️ 외국인 2명과 함께 ^^

**Comment by syoun602** [comment]
# 6인의개발자 회고
클레이: 회고하면서 다른 크루의 의지가 느껴져서 덩달아 힘이 난다! 
병민: 굳굳굳 오랜만에 보니 좋아요 
안: 오랜만에 보라조를 만나서 즐거웠다!!! 
썬: 회식이 기대되네요~ 
봄: 오랜만에 보라 조 만나서 즐겁게 이야기 해서 좋았다~ 
이브: 오랜만이네요 좋아요

[14조 회고 ⭐️](https://docs.google.com/spreadsheets/d/1NkJVFSK7pj75A61i5TnTmy9_OICGTa_wkhW928_l7Mg/edit#gid=0)

## 응 배운거 다 까먹어도 방학하면 그만이야~
![image](https://user-images.githubusercontent.com/67397679/162356723-149500c6-2856-48d4-93de-c2545ad2ca8a.png)

**Comment by Yboyu0u** [comment]
[우짤코조회고](https://docs.google.com/spreadsheets/d/1kbMtPIaQySyxk7GdvEx9bSEFWoG_LGm6ZrFEVmKqY9E/edit#gid=0)

잉: 오랜만에 보이는 라디오랑 진실된 회고를 하게 돼서 재밌고 또 재밌었다.

파랑: 고민한 부분들이 혼자만의 고민이 아니었다고 생각하니 위로가 되네요!

앨버: 회고를 하면서 레벨1때 흩어져있던 생각들을 정리할 수 있어서 좋았다. 오랜만에 보이라 조랑 만나서 재밌었습니다~

아서: 회고하면서 레벨1을 되돌아보고, 오랜만에 보이라조와 만나서 좋았습니다!

아키: Lv1 회고를 하면서 감정을 공유하니 좋네요!

티거: 오랜만에 보라 조 만나서 얘기하니까 좋네요~~~! (는 매일 보던 사람들이긴 함) 레벨 1을 돌아볼 수 있어서 좋았습니다 공감 천 번 만 번!!!

오리: 레벨1동안을 정리하는 시간이 되어서 좋았다

꼬린: 간만에 보이는라디오 조 만나고 이야기 나누어서 좋았습니다~

**Comment by sojukang** [comment]
## 조라쿤(예비(반란)군)
[보라회고링크!!](https://docs.google.com/spreadsheets/d/1TEoELXfQZDyXEYRi9UF2tMPvVYM9wHBTMXs3ChNyb5k/edit#gid=0)

## 각자 남기고 싶은 말👋
- `샐리`: 오랜만에 보이는 라디오 조랑 레벨 1 회고 해서 재밌었다 😆오프라인으로 수업 하는 날이 기대된다.
- `범고래`: 보라팀 오랜만에 모여서 좋았습니다! 일요일에 봐요~!
- `레넌`: 보라 회식이 기대된다. 방학 때 잘 놀고 레벨 2를 준비해야겠다.
- `태태`: 오랜만에 만나서 수다 떠니까 즐겁다. 회식도 기대된다!!! 방학이 일주일밖에 안돼서 아쉽
- `소주캉`: 처음과 끝을 마무리하는 보이는 라디오로 회고하니까 수미상관이 맞아서 좋았다. 
- `라쿤`: 보이는 라디오 조와 걱정 없이 이야기 나누니 좋았다🎉
